python/torch_mlir_e2e_test/torchscript/configs/eager_mode.py

```diff
@@ -0,0 +1,45 @@
+# Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+# Also available under a BSD-style license. See LICENSE.
+
+import torch
+from torch.utils._pytree import tree_map
+
+from torch_mlir.eager_mode.torch_mlir_tensor import TorchMLIRTensor
+from torch_mlir_e2e_test.torchscript.framework import TestConfig, Trace, TraceItem
+
+
+def wrap(e):
+    return TorchMLIRTensor(e.detach().clone()) if isinstance(e, torch.Tensor) else e
+
+
+def unwrap(e):
+    return e.elem.clone() if isinstance(e, TorchMLIRTensor) else e
+
+
+class EagerModeTestConfig(TestConfig):
+    """Trivial test config that exercises eager mode plumbing"""
+
+    def __init__(self):
+        super().__init__()
+
+    def compile(self, program: torch.nn.Module) -> torch.nn.Module:
+        return program
+
+    def run(self, artifact: torch.nn.Module, trace: Trace) -> Trace:
```

Comment:
how long does it take to run the full test suite with this config? Getting this fast enough that it can be running under CI should be our highest-priority goal (in follow-up commits, if necessary).

Reply:
I would say 1-3 minutes would be acceptable.